### PR TITLE
Update KeWaitLocal output to match autofiling requirements

### PR DESCRIPTION
**Pull request by NateD-MSFT**
This updates the KeWaitLocal query to output a message that includes a link to the function where the defect was found, per internal autofiling requirements.

## Checklist for Pull Requests

- [X] Description is filled out.
- [X] Only one query or related query group is in this pull request.
- [X] The version number on changed queries has been increased via the `@version` comment in the file header.
- [X] All unit tests have been run: ([Test README.md](src\drivers\test\README.md)).
- [X] Commands `codeql database create` and `codeql database analyze` have completed successfully.
- [X] A .qhelp file has been added for any new queries or updated if changes have been made to an existing query.

### CODE REVIEW

**Comment by lauragonzalezzz** [comment]
If this is a new requirement for all queries, please update the metadata template table in our wiki.

File: src/drivers/wdm/queries/KeWaitLocal/KeWaitLocal.ql [hunk 1]
```diff
@@ -14,6 +14,9 @@
  * @query-version 1.0
  * @precision high
  * @platform Desktop
+ * @tags correctness
+ *       wddst
```
Comment (lauragonzalezzz) [inline src/drivers/wdm/queries/KeWaitLocal/KeWaitLocal.ql:18]: What is this field? Looks like extra tab.

File: src/drivers/wdm/queries/KeWaitLocal/KeWaitLocal.sarif [hunk 2]
```diff
@@ -6,7 +6,7 @@
       "driver" : {
         "name" : "CodeQL",
         "organization" : "GitHub",
-        "semanticVersion" : "2.11.5",
+        "semanticVersion" : "2.11.6",
```
Comment (lauragonzalezzz) [inline src/drivers/wdm/queries/KeWaitLocal/KeWaitLocal.sarif:9]: Main WDDST is on CodeQL CLI version 2.11.5.

Comment (NateD-MSFT) [reply src/drivers/wdm/queries/KeWaitLocal/KeWaitLocal.sarif:9]: Good catch.  It looks like I have 2.11.5 on my devbox for some reason.  I'll downgrade and re-run.

File: src/drivers/wdm/queries/KeWaitLocal/KeWaitLocal.ql (hunk 1 above)
Comment (NateD-MSFT) [reply src/drivers/wdm/queries/KeWaitLocal/KeWaitLocal.ql:18]: See https://github.com/microsoft-sdl/codeql-queries#required-metadata-fields-for-security-queries.  The extra spacing is coding style for the SDL repo.

**Pull request merged**